src/app/routes/connector-ui/transfer-history-page/transfer-history-page/transfer-history-page.component.html

```diff
@@ -36,51 +36,48 @@
       class="transfer-history-table"
       mat-table
       [dataSource]="transferProcessesList.data.transferProcesses">
-      <ng-container matColumnDef="id">
-        <th *matHeaderCellDef mat-header-cell scope="col">ID</th>
-        <td *matCellDef="let item" mat-cell>{{ item.id }}</td>
+      <ng-container matColumnDef="direction">
+        <th *matHeaderCellDef mat-header-cell scope="col">Direction</th>
+        <td *matCellDef="let item" mat-cell>
+          <mat-icon mat-table-avatar>
```

Comment:
mat-table-avatar is not a directive of angular material, right?

is this copilot generated?

also using one of the utility classes we have like "mat-icon-[30px]" or sth should be preferred over trying to address it via the HTML attribute